notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    model = core.read_model(model=IR_PATH)
</pre><p>could you please add widget for initial device selection like in other notebooks?</p><p> </p><p>(we want to be able check how model works on different devices using ci, unfortunately gradio is not suitable here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Done